constants.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+
+
+START_YEAR = 2010
+
+CORRELATION_THRESHOLDS = {
+    "0": "no correlation",
+    "0.1": "negligible correlation",
+    "0.4": "weak correlation",
+    "0.7": "moderate correlation",
+    "0.9": "strong correlation",
+    "1": "very strong correlation",
+}
+
+TREND_THRESHOLDS = 0.4
+
+
+class NARRATIVE_TEMPLATE(Enum):
+
```

Comment:
FYI  @bhupatiraju 
I am thinking about using these error messages to signal to the users when we have issues showing the narratives. I would love to take your suggestions, but I think we should use the same error message across the app. 
e.g.

![image](https://github.com/user-attachments/assets/0e799268-5ae7-4e7b-85fe-6a42ac8c7786)

Reply:
@bhupatiraju I added the option to feed dataset_name into the data unavailable message.